types/react-bootstrap/lib/NavDropdown.d.ts

```diff
@@ -1,14 +1,16 @@
 import * as React from 'react';
+import { Omit } from 'react-bootstrap';
 import { DropdownBaseProps } from './Dropdown';
 
 declare namespace NavDropdown {
     export interface NavDropdownBaseProps extends DropdownBaseProps {
         active?: boolean;
         noCaret?: boolean;
         eventKey?: any;
+        title: React.ReactNode;
     }
 
-    export type NavDropdownProps = NavDropdownBaseProps & React.HTMLProps<NavDropdown>;
+  export type NavDropdownProps = NavDropdownBaseProps & Omit<React.HTMLProps<NavDropdown>, 'title'>;
```

Comment:
It won't work without `Omit`, right?

Reply:
Yes, without Omit the `title` props type will be combined with one from `HTMLAttributes`, which only accept `string`.

Reply:
Would it be sufficient to swap the order of the intersection, so that the `NavDropdownBaseProps` take precedence?

`export type NavDropdownProps = React.HTMLProps<NavDropdown> & NavDropdownBaseProps`?

Reply:
@vitosamson, I am pretty sure that in Typescript intersection, the order does not matter.

```javascript
type XY = X & Y;
type YX = Y & X;
```

should be equal.

Reply:
Also [`title` is required](https://github.com/react-bootstrap/react-bootstrap/blob/v4-support/src/NavDropdown.js#L20) but the HTML title is not. `Omit` is how we've dealt with this same issue a bunch of places already.